packages/adapter-flagsmith/src/index.ts

```diff
@@ -0,0 +1,84 @@
+import type { Adapter } from 'flags';
+import flagsmith, {
+  IFlagsmithFeature,
+  IInitConfig,
+  IIdentity,
+  createFlagsmithInstance,
+} from 'flagsmith';
```

Comment:
This flagsmith import tries to use react but this is a server library, need at least `flagsmith/next-middleware`